examples/next/faust-nx/pages/sample-page.tsx

```diff
@@ -0,0 +1,26 @@
+import { GetStaticPropsContext } from 'next';
+import { getNextStaticProps } from 'faust-nx';
+import { useQuery, gql } from '@apollo/client';
+import client from 'client';
+
+export default function Page(props: any) {
+  const {data} = useQuery(Page.query);
```

Comment:
We need to use this useQuery to extract the data from the cache

Reply:
I think we'll probably want a React hook or some kind of abstraction here.

Reply:
For this ticket it's not needed as this is how Apollo works by default using SSR,SSG. We might offer an HOC or a custom context but this is something we will have to think how to present it nicely.